.github/workflows/test_web.yml

```diff
@@ -164,10 +164,10 @@ jobs:
     steps:
       - uses: actions/checkout@v5
 
-      - name: Setup Node.js 22
+      - name: Setup Node.js 24
```

Comment:
Not a big deal, but interesting that the browser tests job says the Node.js version in the step name, but  that build for browser tests job does not.